test/rantly_generator_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+require 'rantly/minitest_extensions'
+
+describe Rantly do
+  before do
+    Rantly.gen.reset
+  end
+
+  describe '#range' do
+    it 'returns an integer within range when given two integers' do
+      property_of {
+        i = integer
+        [i,range(i,i)]
+      }.check { |(a,b)|
+        assert_equal a, b
+      }
+      property_of {
+        lo, hi = [integer(100),integer(100)].sort
+        [lo,hi,range(lo,hi)]
+      }.check { |(lo,hi,int)|
```

Comment:
[nitpick] I would use a more explicit name for `int`

Reply:
Renamed it to `num_in_range`.